### ISSUE TITLE
WHD Containers

### ISSUE THREAD

**Issue by shabbirkagalwala** [solarwinds/containers#7]
@leecalcote  This is more of a feature request or a question rather than a bug/issue. 

We noticed that when WHD sends out an email notifying the user about a ticket, the link to the ticket in the email is http instead of https. Is there a flag maybe that we can use or any configuration changes that can be made to have this default to https instead of http? 

To give you some insight, we are using the whd containers behind a proxy which also acts a ssl terminator.

Thank you.

**Comment by shabbirkagalwala** [comment]
@leecalcote  We digged around a bit to check the files and this is how we got it to work.

docker run -d -p 443:443 --name=whd -e URL_HTTPS_PORT=443 -e HTTPS_PORT=443 solarwinds/whd-embedded:latest

**Comment by leecalcote** [comment]
@shabbirkagalwala yes, you got it. I meant to point out #2 earlier. Is this the same issue?

**Comment by shabbirkagalwala** [comment]
@leecalcote  Not really since we have our WHD containers behind a proxy  which was acting as a SSL terminator we were not able to use SSO because the container was listening on 8081 rather than 443. And for SSO to work it requires the meta data url to be https rather than http.  So the above variables made it work.

**Comment by shabbirkagalwala** [comment]
I forgot to mention change the Dockerfile to Expose 443 instead of 8081.
